Apps/Admin/Client/Components/Delegation/DelegateTable.razor.cs

```diff
@@ -59,9 +59,9 @@ private static Color GetStatusColor(DelegationStatus status)
             };
         }
 
-        private void ToggleRemoveDelegate(string hdid, bool value)
+        private void ToggleRemoveDelegate(string hdid, bool? value)
```

Comment:
I've come to hate nullable types. they just litter the code with random checks and now making once simple boolean checks require checking against a boolean value (sigh).... we've lost its binary nature.